packages/react-events/src/Press.js

```diff
@@ -256,7 +250,7 @@ const PressResponder = {
             ) {
               return;
             }
-            // Ignore right-clicks
+            // Ignore middle- and right-clicks
```

Comment:
Typo with the additional `-`

Reply:
Intentional as per https://english.stackexchange.com/questions/113422/how-to-use-hyphens-appropriately-when-listing-multiple-hyphenated-terms